### ISSUE TITLE
Release version supporting Google Terraform provider >=5

### ISSUE THREAD

**Issue by gustaff-weldon** [terraform-google-modules/terraform-google-bootstrap#250]
### TL;DR

Google Terraform provider 5.0.0 was [released a month ago](https://registry.terraform.io/providers/hashicorp/google/5.0.0/docs)

Modules under `terraform-google-modules` still do not provide a tagged and released version supporting provider 5 and above.

https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/v6.4.0/versions.tf#L23

<img width="495" alt="Screenshot 2023-11-09 at 09 18 14" src="https://github.com/terraform-google-modules/terraform-google-bootstrap/assets/411756/01687c6b-c158-404b-9c21-4cd58f916770">

Please release this module (and necessary dependant modules) supporting provider >=5

If this module is not maintained and should not be used, please provide guidance what should be used instead? Is eg. Fabric the one to go now when it comes to Terraforming GCP?

Thanks :)

### Terraform Resources

```markdown
https://registry.terraform.io/providers/hashicorp/google/5.0.0/docs

https://registry.terraform.io/providers/hashicorp/google/latest/docs (5.5)
```

**Comment by gustaff-weldon** [comment]
Looks like this will depend on https://github.com/terraform-google-modules/terraform-google-org-policy/pull/105

**Comment by 3sne** [comment]
Would love to see this merged soon, we are being held back to provider version `<5` because of this module!

**Comment by gustaff-weldon** [comment]
https://github.com/terraform-google-modules/terraform-google-org-policy/tree/v5.3.0 is released 

https://github.com/terraform-google-modules/terraform-google-org-policy/blob/v5.3.0/versions.tf#L23

**Comment by bestefreund** [comment]
Thanks for merging the [related PR](https://github.com/terraform-google-modules/terraform-google-bootstrap/pull/248) for the new Release Version `6.4.1`.
At the momnent the new Tag for this new Release-Version is missing in this Repository and in the Terraform registry.

When will the new version Tag be available?

**Comment by apeabody** [comment]
Hi @bestefreund - Looks like [v6.4.1](https://github.com/terraform-google-modules/terraform-google-bootstrap/releases/tag/v6.4.1) was published on Dec 5, 2023. Cheers!
